package.json

```diff
@@ -13,11 +13,11 @@
     "email" : "me@daniel.gs"
   },
   "dependencies": {
-    "lodash": "3.0.x"
+    "lodash": "^4.17.4"
```

Comment:
I believe it should now be updated to at least `4.17.5`, according to https://nodesecurity.io/advisories/577

Reply:
Correct

Reply:
Caret inclusion takes care of pulling patch and minor releases. Nevertheless, I'll update this pull request in the hopes of eventual merge :crossed_fingers: :wink: